rest-mvc/src/main/java/com/intuit/tank/rest/mvc/rest/controllers/AgentController.java

```diff
@@ -110,6 +111,17 @@ public ResponseEntity<TankHttpClientDefinitionContainer> getClients() {
         return new ResponseEntity<>(agentService.getClients(), HttpStatus.OK);
     }
 
+    @RequestMapping(value = "/availability", method = RequestMethod.POST, consumes = { MediaType.APPLICATION_JSON_VALUE })
+    @Operation(description = "Sets the availability status for a standalone agent", summary = "Set standalone agent availability", hidden = true)
+    @ApiResponses(value = {
+            @ApiResponse(responseCode = "200", description = "Successfully set agent availability"),
+            @ApiResponse(responseCode = "400", description = "Could not update agent availability", content = @Content)
+    })
+    public ResponseEntity<Void> setStandaloneAgentAvailability(
+            @RequestBody @Parameter(description = "Agent availability request to update standalone agent availability", required = true) AgentAvailability availability) {
+        return ResponseEntity.ok().build();
```

Comment:
@Zakaria-Kofiro Not seeing a service call here, is that expected?

Reply:
Added back, deleted on accident while updating response entity, good catch!